strongbox-security/strongbox-authentication-registry/src/test/resources/etc/conf/eudst-strongbox-authentication-providers.yaml

```diff
@@ -0,0 +1,5 @@
+strongbox:
+  authentication:
+    xmlUserDetailService:
+      order: 0
```

Comment:
Do we really need an `order` field here? I mean -- won't the list of authentication providers be a `LinkedList` anyway? Otherwise, this will open a follow-up problem where we'll need to make sure there are no two user details services under the same `order` number, (in case somebody misconfigured it).

Reply:
> Do we really need an order field here? I mean -- won't the list of authentication providers be a LinkedList anyway?

currently this is not a list, have a look at `strongbox-storage-api-resources/src/main/resources/etc/conf/strongbox-authentication-providers.yaml`. 

> Otherwise, this will open a follow-up problem where we'll need to make sure there are no two user details services under the same order number, (in case somebody misconfigured it).

yeah, we can probably check it, but anyway there will be no errors in this case, this will just be treated as undefined order